examples-standalone/grid-live-data/src/app/main-panel/transactions/transactions.component.html

```diff
@@ -7,7 +7,7 @@
                 height="40px"
                 [imageSrc]="getCardImg(card)"
                 shape="circle"
-                style="background-color: transparent"
+                [style.background-color]="'transparent'"
```

Comment:
what is wrong with the previous line? + the syntax of the new one is incorrect :)

Reply:
I thought that the **style** property in HTML needs to be replaced with **class** instead. That is why I decided to not make a separate class for a single CSS property, and I just replace it with the current syntax. If need I could revert it.